src/Mutator/Unwrap/UnwrapArrayReplace.php

```diff
@@ -36,13 +36,37 @@
 namespace Infection\Mutator\Unwrap;
 
 use Generator;
+use Infection\Mutator\Definition;
+use Infection\Mutator\MutatorCategory;
 use PhpParser\Node;
 
 /**
  * @internal
  */
 final class UnwrapArrayReplace extends AbstractUnwrapMutator
 {
+    public static function getDefinition(): ?Definition
+    {
+        return new Definition(
+            <<<'TXT'
+Replaces an `array_replace` function call with its first operand. For example:
```

Comment:
Not only first, see `getParameterIndexes()` and the tests. This definition is not correct.

I didn't check all the cases, but could you please check similar unwrap mutators that produces more than 1 mutations? Definition shouldn't say only "first" for them

Reply:
Done in #962